### ISSUE TITLE
[a11y] Toolbar not accessible in TinyMCE editor

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/joomla-cms#20705]
If you don't use the mouse, if you can't use the touchscreen, you can't use the toolbar in the TinyMCE editor, for example when editing an article. Perhaps I don't know something.
### Steps to reproduce the issue
1. On Backend open e.g page New Article
2. Disconnect the mouse (or not use it)
3. Start writing an article.
4. Try using any of the following buttons on the TinyMCE editor's toolbar
### Expected result
The toolbar in TinyMCE editors gets the focus, and you can select any button or command.
### Actual result
When you use the tab key, the focus will be moved out of the TinyMCE editor.

**Comment by dgrammatiko** [comment]
https://www.tinymce.com/docs/advanced/accessibility/

**Comment by brianteeman** [comment]
Going to close this one. As shown in the provided link tinymce does support keyboard navigation.

This would however be excellent information to be described in the a11y documentation

**Comment by zwiastunsw** [comment]
I knew I didn't know :) Thank you @dgrammatiko . Thank you, @brianteeman. I undertake to describe this in documentation on Joomla Docs. 
Ready:  https://docs.joomla.org/Chunk30:TinyMCE
